.github/workflows/dependabot-ci.yml

```diff
@@ -0,0 +1,78 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+name: dependabot-ci
+on:
+  pull_request:
+    types: [opened, ready_for_review]
+concurrency:
+  group: dependabot-ci-${{ github.event.pull_request.number || github.sha }}
+  cancel-in-progress: ${{ github.event_name == 'pull_request' }}
+jobs:
+  dependabot-check:
+    if: ${{ github.actor == 'dependabot[bot]' && github.event_name == 'pull_request'}}
```

Comment:
dependabot also creates PRs to upgrade Maven dependencies.  This workflow should avoid any further changes if `pnpm-lock.yaml` was not updated by the PR.

Reply:
Hello, I had set the dependabot to only scan for JS dependencies,
```directory: "hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/"```
Shall I then make this change normal? As in it will scan both Java dependencies and JS dependencies?

Reply:
Yes, it should update Java dependencies, too, just like it does currently (without the changes in this PR).

Reply:
Updated in the latest commit.